README.md

```diff
@@ -253,10 +280,13 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://blush.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808594-ada33e0a-022f-4b46-b595-fb816b6954fe.png' alt="Blush_freemium_illustrations" /></a> | [Blush](https://blush.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Small PNG<br>• Medium PNG<br>• Large PNG<br>• SVG<br><br> ✌️Author <br> Blush Design                                                                                         |
+| <a href="https://www.avatarz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194876748-7f29c58a-9d0f-4822-926c-d07a90b5e8f2.png' alt="Avatarz_freemium_illustrations" /></a> | [Avatarz](https://www.avatarz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                 |
 | <a href="https://www.ls.graphics/illustrations/ciank-illustration-kit"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480180-8b180a05-2fd2-42d1-a6e3-88206018d7fd.png' alt="Ciank_freemium_illustration_kit" /></a> | [Ciank](https://www.ls.graphics/illustrations/ciank-illustration-kit)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br> • Sketch<br> • Adobe Illustrator<br> • Affinity<br> • Adobe XD<br> • PNG<br> • SVG<br><br> ✌️Author <br> LS Graphics |
 | <a href="https://www.ls.graphics/illustrations/clak"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808599-e430ada2-18af-4581-bfe7-c70b2b832853.png' alt="Clak_freemium_illustrations" /></a> | [Clak](https://www.ls.graphics/illustrations/clak)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Ls Graphics                                               |
 | <a href="https://characterz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194710390-b436f918-f494-41a0-bda1-76f205e95937.png' alt="make_this_appropriate" /></a> | [CHARACTERZ 2](https://characterz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                                                            |
+| <a href="https://emojiz.design/"><img width='700' src='https://user-images.githubusercontent.com/34191321/194711655-63359e54-bfee-493d-83e1-3b8b14bb4915.png' alt="Emojiz_freemium_illustrations" /></a> | [Emojiz](https://emojiz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                       |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195575740-741a625a-7631-4e4e-9d24-5db0f1f30932.png`